Source/carthage/Build.swift

```diff
@@ -30,6 +30,7 @@ extension BuildOptions: OptionsType {
 			<*> m <| Option(key: "platform", defaultValue: .All, usage: "the platforms to build for (one of 'all', 'macOS', 'iOS', 'watchOS', 'tvOS', or comma-separated values of the formers except for 'all')" + addendum)
 			<*> m <| Option<String?>(key: "toolchain", defaultValue: nil, usage: "the toolchain to build with")
 			<*> m <| Option<String?>(key: "derived-data", defaultValue: nil, usage: "path to the custom derived data folder")
+			<*> m <| Option(key: "cache-builds", defaultValue: true, usage: "use cached builds when possible")
```

Comment:
This should default to `false` for the initial release, I think. Then people can try it out and use it—but if there are bugs, then it won't bite everybody.